incubator/kafka/notes.txt

```diff
@@ -0,0 +1,36 @@
+You can connect to Kafka by running a simple pod in the K8s cluster like this with a configuration like this:
```

Comment:
Helm will specifically look for a while called `NOTES.txt`, render it as a template, and print it out after installing/upgrading or running `helm status` on the release. This will allow you to print out working commands with the name of the release! See  https://github.com/kubernetes/charts/blob/master/incubator/patroni/templates/NOTES.txt for an example :)